test-network-function/networking/suite.go

```diff
@@ -324,3 +348,99 @@ func testNodePort(env *config.TestEnvironment) {
 		}
 	})
 }
+
+func parseVariables(res string, declaredPorts map[key]string) error {
```

Comment:
We should have a unit test surrounding this func.

Reply:
I added - log.Info("The parser is Fail.") and we will return the error, would you prefer that if it failed then the whole test failed and not continue?

Reply:
After the meeting, I close this , I will create new PR that include unit test for all function in file suite.go in networking.